src/Symfony/Component/VarDumper/Tests/Fixtures/Twig.php

```diff
@@ -26,14 +26,11 @@ public function getTemplateName()
 
     public function getDebugInfo()
     {
-        return array (19 => 2);
+        return array(19 => 2);
     }
 
-    public function getSource()
+    public function getSourceContext()
     {
-        return "   foo bar
-     twig source
-
-";
+        return new Twig_Source("   foo bar\n     twig source\n\n", 'foo.twig', 'foo.twig');
```

Comment:
Please also add tests for other cases:

- Twig_Source with an empty code string (i.e. a non-debug compilation)
- Twig_Source without a path (i.e. a loader not based on the filesystem, or a loader not updated to use new features)